ksml/src/test/java/io/axual/ksml/python/PythonContextConfigTest.java

```diff
@@ -0,0 +1,46 @@
+package io.axual.ksml.python;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PythonContextConfigTest {
+
+    private final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+
+    @Test
+    @DisplayName("Builder defaults all flags to false")
+    void defaultBuilderFlags() {
+        var cfg = PythonContextConfig.builder().build();
+        assertFalse(cfg.allowHostFileAccess(),     "allowHostFileAccess should default to false");
```

Comment:
We also have assertJ available.
Using SoftAssertions you can use a more readable assertion, and actually test all values at once.